osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -130,11 +142,27 @@ protected override void LoadComplete()
             valid.BindValueChanged(_ => Scheduler.AddOnce(refresh));
             requiredMods.CollectionChanged += (_, __) => Scheduler.AddOnce(refresh);
 
+            onScreenLoader.DelayedLoadStarted += _ =>
```

Comment:
Unfortunately this drawable is shared with playlists (the mode), and also used in places wherein this fetch doesn't need to happen. For instance it will run on the room creation screen before anything online has happened yet.

The immediate simplest way to counteract this would be to add local guards (`showItemOwner && Item.OwnerID > 0`, `Item.Beatmap.Value != null`). But the time is rapidly approaching wherein this component needs to be split into multiple ones, this one implementation is approaching critical mass wrt what it can do.

Reply:
Hmm, I did test this not to break that scenario. The code path already checks whether the beatmap is already populated and early aborts.

Reply:
Oh, right, the beatmap fetch will be a no-op lower down, sure. I forgot about that. But there's still the issue of the user fetch (happens even if owner id is 0 / owner is not shown on playlist), and in general the fact that playlist item panels in playlists mode are now calling things from `MultiplayerClient` at all is pretty weird.

Reply:
Yeah, for sure. I don't think this is the correct PR to be doing larger refactors there (will probably conflict with @smoogipoo's ongoing work on the queue display).

I'll fix the user lookup part.